src/components/ArticleForm/ArticleForm.jsx

```diff
@@ -6,13 +6,20 @@ import { toast } from 'react-toastify';
 const ArticleForm = ({ onSubmit, loading, countryUrl, continentUrl }) => {
     const [formErrors, setFormErrors] = useState({});
     const today = new Date();
-    today.setDate(today.getDate() - 1);
+
     const [form, setForm] = useState({
         country: '',
-        date: today.toISOString().split('T')[0],
+        date: getStartDayOftheWeek(today),
         access: 'all-access',
     });
 
+    function getStartDayOftheWeek(date) {
+        const diff = date.getDate() - date.getDay() + (date.getDay() === 0 ? -6 : 1);
+
+        const startOfWeek = new Date(date.setDate(diff));
+        return startOfWeek.toISOString().split('T')[0];
+    }
+
     useEffect(() => {
         const getLocation = async (lat, lon) => {
             try {
```

Comment:
@ShadowFax1731, as mentioned in the issue #230, we need this feature on the 'Top Africa page` (for now) !
So please, use it only on the 'Top Africa' page